src/main.js

```diff
@@ -37,6 +42,33 @@ async function run() {
         body += `\n:no_entry: ${errorMessage}`;
       }
 
+      const updateGitHubComment = commentId =>
+      octokit.issues.updateComment({
+        repo: github.context.repo.repo,
+        owner: github.context.repo.owner,
+        comment_id: commentId,
+        body,
+      })
+
+      if (updateComment == "true") {
+        const issueComments = await octokit.issues.listComments({
+          repo: github.context.repo.repo,
+          owner: github.context.repo.owner,
+          issue_number: github.context.payload.pull_request.number,
+        })
+
+        const existingComment = issueComments.data.find(comment =>
+          comment.body.includes(commentIdentifier(process.env.GITHUB_WORKFLOW)),
+        )
+
+        if (existingComment) {
+          console.log('Update Comment ID: ' + existingComment.id);
+          await updateGitHubComment(existingComment.id);
+          return
+        }
+        console.log('Comment does not exist, create a new one');
+      }
+
       await octokit.issues.createComment({
```

Comment:
This should be within an else block right? That way this wont not executed when `updateComment` is `true`.